From e49d6ba388648bbae71abb18dbcb507fb2f2f047 Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@emb.dev>
Date: Mon, 8 Apr 2024 16:13:14 +0200
Subject: [PATCH] drivers: net: nsos: fix error message on EPOLL_CTL_DEL error

Fix a copy-pasted error message to reference proper epoll_ctl() operation.

Signed-off-by: Marcin Niestroj <m.niestroj@emb.dev>
---
 drivers/net/nsos_adapt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/nsos_adapt.c b/drivers/net/nsos_adapt.c
index a7ecd46eeb4..9c8184bd61b 100644
--- a/drivers/net/nsos_adapt.c
+++ b/drivers/net/nsos_adapt.c
@@ -526,7 +526,7 @@ void nsos_adapt_poll_remove(struct nsos_mid_pollfd *pollfd)
 
 	err = epoll_ctl(nsos_epoll_fd, EPOLL_CTL_DEL, pollfd->fd, NULL);
 	if (err) {
-		nsi_print_error_and_exit("error in EPOLL_CTL_ADD: errno=%d\n", errno);
+		nsi_print_error_and_exit("error in EPOLL_CTL_DEL: errno=%d\n", errno);
 		return;
 	}
 
-- 
2.45.2

